generators/app/templates/_karma.conf.js

```diff
@@ -18,7 +18,8 @@ function listFiles() {
       path.join(conf.paths.src, '/app/**/*.js'),
       path.join(conf.paths.src, '/**/*.spec.js'),
       path.join(conf.paths.src, '/**/*.mock.js'),
-      path.join(conf.paths.src, '/**/*.html')
+      path.join(conf.paths.src, '/**/*.html'),
+      path.join(conf.paths.bower, '/**/*.min.js')
```

Comment:
this is tricky -- not all bower distributions have `.min` files. Bower specifically has a configuration entry to describe this: https://github.com/bower/bower.json-spec (see "main")